<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushDialog</class>
 <widget class="QDialog" name="BrushDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>CapStyle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_capstyle">
           <item>
            <property name="text">
             <string>FlatCap</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SquareCap</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RoundCap</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Join Style</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_joinstyle">
           <item>
            <property name="text">
             <string>BevelJoin</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RoundJoin</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MiterJoin</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>LineStyle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_linestyle">
           <item>
            <property name="text">
             <string>SolidLine </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DashLine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DotLine    </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DashDotLine   </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DashDotDotLine</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
